docs/staking/intro.rst

```diff
@@ -0,0 +1,312 @@
+for sending various transactions and other types of interaction with a running chain.
+The rest of this tutorial will cover several commands from ``gaia client`` and assume
+that you've setup or joined a testnet.
+
+Generating Keys
+---------------
+
+First, let's generate a key named ``alice``:
+
+::
+
+    gaia client keys new alice
+
+This will output:
+
+::
+
+    Enter a passphrase:
+    Repeat the passphrase:
+    alice	    5D93A6059B6592833CBC8FA3DA90EE0382198985
+    **Important** write this seed phrase in a safe place.
+    It is the only way to recover your account if you ever forget your password.
+
+    inject position weather divorce shine immense middle affair piece oval silver type until spike educate abandon
+
+which has your address and will be re-used throughout this tutorial.
+We recommend doing something like ``MYADDR=<your address>``. Writing 
+down the recovery phrase is crucial for production keys, however,
+for this tutorial you can skip this step.
+
+Because ``alice`` will be the initial validator, we need another key, ``bob`` who will first receives tokens from ``alice``, then declare candidacy as a validator. We also need an account for ``charlie`` who will bond and unbond to ``bob``.
+
+::
+
+    gaia client keys new bob
+    gaia client keys new charlie
+
+Now we can see the keys we've created:
+
+::
+
+    gaia client keys list
+
+which shows something like:
+
+::
+
+    All keys:
+    alice           5D93A6059B6592833CBC8FA3DA90EE0382198985
+    bob             5A35E4CC7B7DC0A5CB49CEA91763213A9AE92AD6
+    charlie         48F74F48281C89E5E4BE9092F735EA519768E8EF
+
+Try adding the ``--output json`` flag to the above command to get more information.
+The information for these keys is stored in ``~/.cosmos-gaia-cli``.
+We've got our keys made, so let's move on to the next step.
+
+Sending Tokens
+--------------
+
+We'll have ``alice`` who is currently quite rich, send some ``fermions`` to ``bob``:
+
+::
+
+    gaia client tx send --amount=1000fermion --sequence=1 --name=alice --to=5A35E4CC7B7DC0A5CB49CEA91763213A9AE92AD6
+
+where the ``--sequence`` flag is to be incremented for each transaction, the ``--name`` flag names the sender, and the ``--to`` flag takes ``bob``'s address. You'll see something like:
+
+::
+
+    Please enter passphrase for alice: 
+    {
+      "check_tx": {
+        "gas": 30
+      },
+      "deliver_tx": {
+        "tags": [
+          {
+            "key": "height",
+            "value_type": 1,
+            "value_int": 2963
+          },
+          {
+            "key": "coin.sender",
+            "value_string": "5D93A6059B6592833CBC8FA3DA90EE0382198985"
+          },
+          {
+            "key": "coin.receiver",
+            "value_string": "5A35E4CC7B7DC0A5CB49CEA91763213A9AE92AD6"
+          }
+        ]
+      },
+      "hash": "423BD7EA3C4B36AF8AFCCA381C0771F8A698BA77",
+      "height": 2963
+    }
+
+Check out ``bob``'s account, which should now have 992 fermions:
```

Comment:
"should now have 1000 fermions"

Reply:
or rather 7 tokens if we scale down the tutorial

Reply:
i'll just add a note that the tokens are arbitrary in the tut